### PR TITLE
[Entry] ClickSpeed

### PR DESCRIPTION
**Pull request by aindong**
Adjust the volume base on the Click / Second

### CODE REVIEW

File: src/entries/ClickSpeed/ClickSpeed.jsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      setClickPerS(cps)
+    }
+
+    if (activeTime > 1000) {
+      const mCPS = Math.max(clicksPerS, maxCPS)
+      setMaxCPS(mCPS)
+    }
+
+    changeVolume(clicksPerS * 10)
+    console.log("Volume", clicksPerS * 10)
```
Comment (fransaycon) [inline src/entries/ClickSpeed/ClickSpeed.jsx]: remove the consoles bro :D

File: src/entries/ClickSpeed/ClickSpeed.jsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    console.log("Idle Time", idleTime)
+    console.log("Clicks: ", normalize(clicks))
+    console.log("CPS: ", normalize(clicksPerS))
+    console.log("Max CPS: ", normalize(maxCPS))
+    console.log("Time: ", normalize(activeTime / 1000))
+  }
+
+  const mouseUp = () => {
+    const newEndTime = getCurrentTime()
+    console.log("End Time", newEndTime)
```
Comment (fransaycon) [inline src/entries/ClickSpeed/ClickSpeed.jsx]: stray console.

**Review by fransaycon — CHANGES_REQUESTED** [review]
thanks for the entry. just a few comments. :D

**Comment by aindong** [comment]
Thank you @franreysaycon! I totally forgot about my consoles! 😅

**Review by fransaycon — APPROVED** [review]

**Pull request merged**